semanticdb/semanticdb3/javacp.md

```diff
@@ -0,0 +1,551 @@
+### Java
+
+In this section, we exhaustively map Java language features onto SemanticDB.
+As a reference, we use the Java Language Specification [\[85\]][85] (referred
+to as "JLS" in the text below).
+
+<a name="java-symbol"></a>
+#### Symbol
+
+In this section, we describe the Java symbol format.
+
+<table>
+  <tr>
+    <td><b>Symbols</b></td>
+    <td><b>Format</b></td>
+  </tr>
+  <tr>
+    <td>
+      Global symbols
+      <a href="#symbol">↑</a>
+    </td>
+    <td>
+      <ul>
+        <li>
+          For <a href="#java-root-package">root package</a>, its descriptor.
+        </li>
+        <li>
+          For unnamed packages <a href="https://docs.oracle.com/javase/specs/jls/se8/html/jls-7.html#jls-7.4.2">[21]</a>,
```

Comment:
I understand that there can be several unnamed packages (was surprised to learn about that), but I think this still should be singular for consistency with other bullet points.